site/app/templates/EditPronounsForm.twig

```diff
@@ -10,22 +10,24 @@
     </p>
     <div class="flex-col" id="user_settings_modal">
         <input id="user-pronouns-csrf" type="hidden" name="csrf_token" value="{{ csrf_token }}" />
-        <h2>My Preferred Pronouns</h2>
+        <h2>My Pronouns</h2>
         <div class="edit_user_row">
-            <label for="user-pronouns-change">Preferred Pronouns:&emsp;</label>
+            <label for="user-pronouns-change">Pronouns:&emsp;</label>
             <input type="text"
                    name="user_pronouns_change"
                    value="{{ user_pronouns }}"
                    id="user-pronouns-change"
                    list="default_pronouns_list"
-                   placeholder="Type Custom Or Select">
+                   placeholder="Type Or Select"
+                   size="30">
                 <datalist id="default_pronouns_list">
                     <option value="He/Him"></option>
                     <option value="She/Her"></option>
                     <option value="They/Them"></option>
-                    <option value="Custom"></option>
+                    <option value="(Type Custom Here)"></option>
                 </datalist>
             </input>
+            <a href="javascript:clearPronounsBox();"><i class="fas fa-trash" style="padding:0.5rem;"></i></a>
```

Comment:
Typically we should not be using `a` tags for non navigation purposes since they can mess with screen readers and act a little weird on certain browsers 

This should be a button wrapping the icon and you can remove the styling for it so only the icon shows up
```html
<button aria-label="Clear pronoun input" type="button" style="background: none;border:none;" onclick="clearPronounsBox();"><i> icon... </i></button
```

The aria-label will prevent the accessibility checker from complaining about a button with no text and the style removes any bordering. The type prevents the button from submitting the form 

You could also optionally add a title field to the icon, this will show some help text when a user hover's over it, something like 

```html
<i title="Clear pronouns field" ></i>
```

Here's some additional context on the subject
https://css-tricks.com/buttons-vs-links/
https://fontawesome.com/docs/web/dig-deeper/accessibility
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#accessibility

Reply:
Done!